### DISCUSSION TITLE
Could you support Top Fourth?

### DISCUSSION THREAD

**Discussion by wilsonssss** [rxhanson/Rectangle#918] [Ideas]
![wohi2o5YlO](https://user-images.githubusercontent.com/4098951/189568280-213e0762-12df-4961-852c-4e3f85481d06.jpg)

**Comment by rxhanson** [comment]
First fourth will be executed as top fourth on a portrait display. Is that not working for you?

**Reply by wilsonssss** [reply]
That’s impossible bro. Pls focus on the preview icon, it’s horizontal, not like vertical at all.
![4A3B30FC-54B7-4848-B3AC-15E6F4E9A9DB](https://user-images.githubusercontent.com/4098951/189696075-75b329dc-4bd9-4790-b82d-8dd230256d03.jpeg)
